pkg/cmd/server/server.go

```diff
@@ -271,3 +284,10 @@ func (c *completedServerConfig) Run(ctx context.Context) error {
 
 	return nil
 }
+
+type GrpcMetadataCredentials map[string]string
```

Comment:
Is this being used?

Reply:
good catch, I missed that when I was cleaning up